app/import/robot/processor.js

```diff
@@ -0,0 +1,45 @@
+const { createCdo } = require('../../repos/cdo')
+const sequelize = require('../../config/db')
+const { addYears } = require('date-fns')
+
+const processRegister = async (register) => {
+  await sequelize.transaction(async (t) => {
+    for (const record of register.add) {
+      const owner = record.owner
+
+      owner.phoneNumber = `0${owner.phoneNumber}`
+
+      const data = {
+        owner: {
+          ...owner,
+          primaryTelephone: owner.phoneNumber
+        },
+        dogs: record.dogs.map(d => ({
+          ...d,
+          source: 'Robot',
+          breed: 'XL Bully',
+          enforcement: {
+            cdoIssued: null,
+            cdoExpiry: null,
+            policeForce: 1
+          },
+          insurance: {
+            company: 'Dogs Trust',
+            renewalDate: addYears(d.insuranceStartDate, 1)
+          }
+        })),
+        enforcementDetails: {
```

Comment:
Not sure why we have enforcement at two different levels. Probably not an issue as will only get created at one level in DB?